aws_embedded_metrics/logger/metrics_context.py

```diff
@@ -176,3 +177,21 @@ def create_copy_with_context(self, preserve_dimensions: bool = False) -> "Metric
     @staticmethod
     def empty() -> "MetricsContext":
         return MetricsContext()
+
+    def set_timestamp(self, timestamp: datetime) -> None:
```

Comment:
You have created the `set_timestamp` method in metrics context but forgot to create it in metrics logger. Without this change customers will not be able to use it.

Reply:
Thanks for pointing this out. Its added to MetricsLogger class and covered with unit test and integration test.